app/src/main/java/com/cmput301f20t21/bookfriends/ui/browse/BrowseFragment.java

```diff
@@ -0,0 +1,69 @@
+package com.cmput301f20t21.bookfriends.ui.browse;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.SearchView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentContainerView;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentTransaction;
+import androidx.lifecycle.ViewModelProvider;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.cmput301f20t21.bookfriends.R;
+
+public class BrowseFragment extends Fragment {
+
+    private BrowseViewModel mViewModel;
+
+    private RecyclerView recyclerView;
+        private RecyclerView.Adapter mAdapter;
```

Comment:
Weird indentation 😄

Reply:
oops

Reply:
since we're not following the convention of naming private field with prefix "m", we should start renaming these default ones.